webapp/src/main/webapp/src/app/sender/archive/search-event.ts

```diff
@@ -0,0 +1,9 @@
+export class SearchEvent {
+
+	startDate?: string = '';
```

Comment:
Do you really need to initialize them to blank strings?

Reply:
Just to avoid the send of null value to backend.

Reply:
Null values are okay,I see no point in using blanks